ZocBuild.Database.Application/MainWindow.xaml.cs

```diff
@@ -111,7 +112,7 @@ private void btBuild_Click(object sender, RoutedEventArgs e)
                     var fileSystem = new FileSystem();
                     var db = dbSetting.Create(connection, transaction);
                     var gitProcess = new ExternalProcess(pathToGit.FullName);
-                    var dvcsScriptRepo = new GitScriptRepository(dbSetting.ScriptsPath, dbSetting.ServerName, dbSetting.DatabaseName, gitProcess, fileSystem, sqlParser, false);
+                    var dvcsScriptRepo = new GitScriptRepository(dbSetting.ScriptsPath, dbSetting.ServerName, dbSetting.DatabaseName, gitProcess, fileSystem, sqlParser, new ConsoleLogger(SeverityLevel.Warning), false);

```

Comment:
Would the WpfLogger need a visual component also? Assuming they launch the sample application from the command line they would be able to view the output from the console logger